.vscode/launch.json

```diff
@@ -36,6 +36,17 @@
                 "ENABLE_PACKAGEVARIANTSETS": "true"
             }
         },
+        {
```

Comment:
Although these are useful, I am hesitant to add more local config to the vscode setup.
All of these "runs" require some form of prerequisite. ie, some/all of porch running in the background.
If we do decide to keep and maintain them, we should prob document somewhere how to use them.
We could either do it inline here, or add it to the docs, which doesn't really make sense as they live in a separate repo (which is wrong imho).

Reply:
no problem. i will add some documentation inline

Reply:
amended the commands with a bit of description let me know if this is not enough detail or not clear enough or if you think these should be moved into documentation etc

Reply:
spot on. thanks